src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -1340,6 +1386,57 @@
         "fallbackVariableName": "presetNavigationDefault"
       }
     },
+    "presetAuthenticationDefault": {
+      "type": "generated",
+      "generator": "switch",
+      "datatype": "string",
+      "parameters": {
+        "evaluator": "C++",
+        "cases": [
+          {
+            "condition": "(preset == 'recommended')",
+            "value": "none"
+          },
+          {
+            "condition": "(preset == 'blank')",
+            "value": "none"
+          }
+        ]
+      }
+    },
+    "authenticationEvaluator": {
+      "type": "generated",
+      "generator": "coalesce",
+      "parameters": {
+        "sourceVariableName": "authentication",
+        "fallbackVariableName": "presetAuthenticationDefault"
+      }
+    },
```

Comment:
I would tend to think this is easier to remove and just provide a default value for authentication... unless we need a different default value for recommended vs blank at some point